### DISCUSSION TITLE
install docear failed

### DISCUSSION THREAD

**Discussion by pdeclarens** [Homebrew/discussions#2269] [Casks]
Hello,

I followed the instructions to install Docear.

Install Docear on Mac OSX
July 25, 2021 Mac App Store	
About the App

    App name: Docear
    App description: docear (App: Docear.app)
    App website: http://docear.org

Install the App

    Press Command+Space and type Terminal and press enter/return key.
    Run in Terminal app:
    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" 2> /dev/null
    and press enter/return key. If you are prompted to enter your Mac's user password, enter it (when you type it, you wont see it on your screen/terminal.app but it would accept the input; this is to ensure no one can see your password on your screen while you type it. So just type password and press enter, even if you dont see it on your screen). Then wait for the command to finish.
    Run:
    brew install docear

Done! You can now use Docear

Unfortunalty installation failed.
message : Warning: No available formula or cask with the name "docear". Did you mean docker?
Where is the catch?

thanks

**Comment by SMillerDev** [comment]
https://docear.org/software/download/ states that they no longer support docear so I expect that it was removed from homebrew because it was broken.

**Comment by pdeclarens** [comment]
Hello,

Thanks for your answer.

Yes I know the stopped development, but the product is top for academic
research. I use it on windows but need to install it on Mac (BigSur 11.6)

I have been able to download the program for mac but when I try to open it
I have a message "unauthirised" although I am admin...

If you have a solution I would be gratefull

regards

Le dim. 10 oct. 2021 à 09:30, Sean Molenaar ***@***.***> a
écrit :

> https://docear.org/software/download/ states that they no longer support
> docear so I expect that it was removed from homebrew because it was broken.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/discussions/discussions/2269#discussioncomment-1454310>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5MJX5XMG6SJK6CW6FNP3LUGE6HLANCNFSM5FWDYUDA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Pascal de Clarens
06 07 85 22 54

**Reply by carlocab** [reply]
Right-click on the app icon and then click "Open".

**Comment by SMillerDev** [comment]
Maybe there is some sort of docear forum where you can ask this? Because there is no homebrew involved in any part of your question, so this isn't the best place to ask.

**Comment by pdeclarens** [comment]
Thanks for the info.
Unfortunately I already tried this with no success.

best regards

Le dim. 10 oct. 2021 à 10:49, Carlo Cabrera ***@***.***> a
écrit :

> Right click on the app icon and then click "Open".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/discussions/discussions/2269#discussioncomment-1454456>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5MJXY72VD6C6Y4JNC65A3UGFHRFANCNFSM5FWDYUDA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Pascal de Clarens
06 07 85 22 54

**Reply by carlocab** [reply]
I'm afraid you'll need to follow Sean's advice here then: https://github.com/Homebrew/discussions/discussions/2269#discussioncomment-1454426
